vendor/bat-native-ads/src/bat/ads/internal/tab_manager/tab_manager_unittest.cc

```diff
@@ -84,7 +85,10 @@ TEST_F(BatAdsTabManagerTest, OpenedNewTabUserActivityEvent) {
   event.created_at = Now();
   expected_events.push_back(event);
 
-  EXPECT_EQ(expected_events, events);
+  const bool is_equal =
+      std::equal(expected_events.begin(), expected_events.end(), events.begin(),
+                 events.end());
+  EXPECT_TRUE(is_equal);
```

Comment:
nit: can use `IsEqualContainers` in `container_util.h`

Reply:
Fixed